typescript2/pkg-playground/src/styles/playground.css

```
@@ -0,0 +1,198 @@
+/* ═══════════════════════════════════════════════════════════════════════
+ * playground.css — Single source of truth for playground design tokens
+ *
+ * Both app-promptfiddle and app-vscode-webview import this file.
+ * All playground components in pkg-playground use these tokens.
+ * ═══════════════════════════════════════════════════════════════════════ */
+
+/* ── Scan pkg-playground component source for Tailwind classes ──────── */
+@source "../**/*.tsx";
+
+/* ── Animations for shadcn dialog/tooltip transitions ──────────────── */
+@import "tw-animate-css";
```

Stylelint (17.5.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)